src/util/globals.js

```diff
@@ -218,7 +218,7 @@ export const getMapTilesSettings = () => {
     return getExtension("mapTiles");
   }
   /* defaults */
-  const api = 'https://api.mapbox.com/styles/v1/trvrb/ciu03v244002o2in5hlm3q6w2/tiles/256/{z}/{x}/{y}?access_token=pk.eyJ1IjoidHJ2cmIiLCJhIjoiY2tqcnM5bXIxMWV1eTJzazN2YXVrODVnaiJ9.7iPttR9a_W7zuYlUCfrz6A';
+  const api = 'https://api.mapbox.com/styles/v1/trvrb/ciu03v244002o2in5hlm3q6w2/tiles/256/{z}/{x}/{y}?access_token=pk.eyJ1IjoidHJ2cmIiLCJhIjoiY2l1MDRoMzg5MDEwbjJvcXBpNnUxMXdwbCJ9.PMqX7vgORuXLXxtI3wISjw';
```

Comment:
what is this key? is it the same key that we're using for other maps? if so, let's set it up as a secret/env var like for the other maps?

Reply:
this is the key from nextstraint

Reply:
we shouldn't be using their public key, we should use ours

Reply:
see how it's set up for portal: https://github.com/uc-cdis/cloud-automation/blob/1f9cbb52cecf449a5ad76f984fe0dda449d924d3/kube/services/portal/portal-deploy.yaml#L162-L168